bin/stripe

```diff
@@ -159,6 +159,14 @@ class CustomerClient(CreateableAPIResourceClient, UpdateableAPIResourceClient,
     url = self.instance_url() + '/subscription'
     self.logged_curl('delete', url, params)
 
+class DisputeClient(CreateableAPIResourceClient, ListableAPIResourceClient,
```

Comment:
This file is deprecated so I'd be inclined to just stop adding new things to it unless @kyleconroy has strong feels otherwise.